site/en/userGuide/tools/cli_commands.md

```diff
@@ -9,164 +9,144 @@ Milvus Command-Line Interface (CLI) is a command-line tool that supports databas
+<h3 id="create-database">Syntax</h3>
 
-Supported metric type. Default is "L2" (L2, IP, HAMMING, TANIMOTO) [L2]:
-L2
+```shell
+create database -db (text) 
+```
 
-sqrt [False]: True
+### Options
 
-Timeout(optional) []:
+| Option | Full name  | Description                                         |
+| :----- | :--------- | :-------------------------------------------------- |
+| -db    | --database | [Required] The database name in milvus.              |
+| --help | n/a        | Displays help for using the command.                |
 
-======
-Return type:
-Assume the vectors_left: L_1, L_2, L_3
-Assume the vectors_right: R_a, R_b
-Distance between L_n and R_m we called "D_n_m"
-The returned distances are arranged like this:
-[[D_1_a, D_1_b],
-[D_2_a, D_2_b],
-[D_3_a, D_3_b]]
+### Examples
 
-Note: if some vectors do not exist in collection, the returned distance is "-1.0"
-======
+#### Example 1
 
-Result:
+The following example create the database <code>testdb</code>  in milvus.
 
-[[3.625464916229248, 3.234992742538452, 3.568333148956299, 3.694913148880005], [2.556027889251709, 2.8901233673095703, 3.385758399963379, 3.3239054679870605]]
+```shell
+milvus_cli > create database -db testdb
 ```
 
-## calc mkts_from_hybridts
+## use Database
 
-Generates a hybrid timestamp based on an existing hybrid timestamp, timedelta, and incremental time interval.
+Use Database in Milvus
 
-<h3 id="calc-mkts_from_hybridts">Syntax</h3>
+<h3 id="use-database">Syntax</h3>
 
 ```shell
-calc mkts_from_hybridts -h (int) -m (float)
+use database -db (text) 
 ```
 
-<h3 id="calc-mkts_from_hybridts">Options</h3>
+### Options
 
-| Option | Full name      | Description                                                                                                                               |
-| :----- | :------------- | :---------------------------------------------------------------------------------------------------------------------------------------- |
-| -h     | --hybridts     | The original hybrid timestamp used to generate a new hybrid timestamp. A non-negative integer that ranges from 0 to 18446744073709551615. |
-| -m     | --milliseconds | The incremental interval in milliseconds.                                                                                                 |
-| --help | n/a            | Displays help for using the command.                                                                                                      |
+| Option | Full name  | Description                                         |
+| :----- | :--------- | :-------------------------------------------------- |
+| -db    | --database | [Required] The database name in milvus.              |
+| --help | n/a        | Displays help for using the command.                |
 
-## calc mkts_from_unixtime
+### Examples
 
-Generates a hybrid timestamp based on the Unix Epoch time, timedelta, and incremental time interval.
+#### Example 1
```

Comment:
Applying the level-4 heading style for labeling examples repeatedly is unnecessary. If there is only one example, it can be omitted. However, if there are multiple examples, using an unordered list would be more appropriate.